<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogDispWindow</class>
 <widget class="QDialog" name="dialogDispWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>497</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>497</width>
    <height>249</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>72</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_WindowSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Window size</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_x0">
      <property name="text">
       <string>x0:</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_x0</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_x0">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_y0">
      <property name="text">
       <string>y0:</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_y0</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_y0">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_xsize">
      <property name="text">
       <string>x size:</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_xsize</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_xsize">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_ysize">
      <property name="text">
       <string>y size:</string>
      </property>
      <property name="buddy">
       <cstring>spinBox_ysize</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_ysize">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_ShowImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Show image</string>
   </property>
   <widget class="QLabel" name="label_imgend">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>24</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>99</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imgshow">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>24</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image #:</string>
    </property>
    <property name="buddy">
     <cstring>horizontalSlider</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_imgstart">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>24</y>
      <width>8</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>24</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>42</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>12</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_WindowSetting">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Window settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_period">
      <property name="text">
       <string>Cosine period:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="buddy">
       <cstring>spinBox_period</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_period">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_phaseshifts">
      <property name="text">
       <string># phase shifts:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="buddy">
       <cstring>comboBox_phaseshifts</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_phaseshifts">
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_color">
      <property name="text">
       <string>Color:</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_color</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_color">
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>white</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_orientation">
      <property name="text">
       <string>Orientation:</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_orientation</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_orientation">
      <item>
       <property name="text">
        <string>vertical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>horizontal</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_gamma">
      <property name="text">
       <string>Gamma correction:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="buddy">
       <cstring>checkBox_gamma</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkBox_gamma">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_x0</tabstop>
  <tabstop>spinBox_y0</tabstop>
  <tabstop>spinBox_xsize</tabstop>
  <tabstop>spinBox_ysize</tabstop>
  <tabstop>spinBox_period</tabstop>
  <tabstop>comboBox_phaseshifts</tabstop>
  <tabstop>comboBox_color</tabstop>
  <tabstop>comboBox_orientation</tabstop>
  <tabstop>checkBox_gamma</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>pushButtonSet</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogDispWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogDispWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
